app/src/main/java/com/keylesspalace/tusky/usecase/TimelineCases.kt

```diff
@@ -130,6 +131,10 @@ class TimelineCases @Inject constructor(
     fun pin(statusId: String, pin: Boolean): Single<Status> {
         // Replace with extension method if we use RxKotlin
         return (if (pin) mastodonApi.pinStatus(statusId) else mastodonApi.unpinStatus(statusId))
+            .onErrorResumeNext(::convertError)
+            .doOnError { e ->
+                Log.w("Failed to pin", e)
```

Comment:
should the log reflect the action (pin/unpin)

Reply:
Not necessarily. How about "Failed to change pin state"

Reply:
Updated the message, also swapped it around to log the original error instead of the converted one since that's probably more useful.